### PR TITLE
Yd/open source

### PR DESCRIPTION
**Pull request by rickyvetter**
Pulling for easier comments.

### CODE REVIEW

File: opensource.html [hunk 1]
```diff
@@ -1,28 +1,121 @@
...
+https://github.com/18F/open-source-policy
+https://engineering.twitter.com/opensource/code-of-conduct
 
-<pre><code>- when to make code public? (in versions or as we go)
-- who owns the copyright?
+<h2> 1.When to make code public? </h2>
+<ul>
+ 	<li>Code will be public when there is structure laid out for rest of the project.</li>
+ 	<li>An empty repo or an ambiguous project should not be public. </li>
+ 	<li>Code should not be public unless there is very good documentation </li>
```
Comment (rickyvetter) [inline opensource.html]: Even if a project is in very early stages?

File: opensource.html [hunk 2]
```diff
@@ -1,28 +1,121 @@
...
+</ul>
+<h2> 2.Who owns the copyright?</h2>
+<p>https://developers.google.com/open-source/cla/corporate</p>
```
Comment (rickyvetter) [inline opensource.html]: Does this mean we should have a cla?

File: opensource.html [hunk 3]
```diff
@@ -1,28 +1,121 @@
...
+		<li>Fork the repo and create your branch from master.</li>
+		<li>If you've added code that should be tested, add tests!</li>
+		<li>If you've changed APIs, update the documentation.</li>
+		<li>Ensure the test suite passes (relavant test suite).</li>
+		<li>If you haven't already, complete the CLA.(big corporations do this)</li>
+	</ul>
+<h2> how? </h2>
+<ul>
+	<li>Socialtables engineer could spent 3 hours a week working an open source project under socialtables repo</li>
+	<li>Projects could either benefit socialtables or other technologies</li>
```
Comment (rickyvetter) [inline opensource.html]: I think there should be more definition here. Maybe something like "Projects worked on for Social Tables impact Social Tables technologies or have a strong possibility of doing so." or something like that.

**Comment by rickyvetter** [comment]
I think this is a really good start! I'd like to be able to sit down with the Ubuntu CoC and form it into something that we all have agreed upon and I'd like to see more points from our discussion in here. @ecgeiser you have some awesome notes from this, right?

**Comment by ecgeiser** [comment]
@rickyvetter my notes from the open source portion are more general and don't really cover the discussion.  I have notes on the activity, but I don't think those would be useful for this purpose.

**Comment by mcwhittemore** [comment]
Looks like we failed to get this mereged in. Going to close it now as its a really old PR.

**Pull request closed without merging**